Formula/pmd.rb

```diff
@@ -11,10 +11,39 @@ class Pmd < Formula
     sha256 "e5ed61e35b5c980b0e1bcf616bc2615f1c5310ad66ce8f4901ca225764e7e78c" => :mavericks
   end
 
+  depends_on :java => "1.8+"
   depends_on "maven" => :build
 
   def install
-    ENV.java_cache
+    java_user_home = buildpath/"java_user_home"
+    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{buildpath}/java_user_home"
+    java_cache_repo = HOMEBREW_CACHE/"java_cache/.m2/repository"
+    java_cache_repo.mkpath
+    (java_user_home/".m2").install_symlink java_cache_repo => "repository"
```

Comment:
You can omit the `=> "repository"` as the base names of the source and target of the symlink are identical.